lib/l10n/timeago/pl.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:timeago/timeago.dart';
+
```

Comment:
dot at the end of every abbreviation

https://sjp.pwn.pl/zasady/Kropka-po-skrocie-wyrazu;629743.html